api/pkg/db/model/model.go

```diff
@@ -63,7 +70,8 @@ type (
 		Categories []*Category `gorm:"many2many:resource_categories;"`
 		CatalogID  uint
 		Versions   []ResourceVersion
-		Tags       []*Tag `gorm:"many2many:resource_tags;"`
+		Tags       []*Tag      `gorm:"many2many:resource_tags;"`
+		Platforms  []*Platform `gorm:"many2many:resource_platforms;"`
```

Comment:
Could you please add `OnDelete:CASCADE` constraints as well

Reply:
@pratap0007 constraints we added. Could you please add `lgtm` if it looks ok to you?